config/initializers/contributors.rb

```diff
@@ -0,0 +1,12 @@
+class Contributors
+  def self.read_contributors
```

Comment:
Make this function return a string rather than a list (since otherwise the same string computation will be repeated every time a user loads the "about" page).

Also use [`Array#join`](https://ruby-doc.org/core-3.1.0/Array.html#method-i-join) to avoid the trailing comma.